pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1577,6 +1576,18 @@ public static class PlanVersions {
     public static final String KEY_OF_MULTISTAGE_EXPLAIN_INCLUDE_SEGMENT_PLAN =
         "pinot.query.multistage.explain.include.segment.plan";
     public static final boolean DEFAULT_OF_MULTISTAGE_EXPLAIN_INCLUDE_SEGMENT_PLAN = false;
+
+    /// Max number of rows operators stored in the op stats cache.
+    /// Although the cache stores stages, each entry has a weight equal to the number of operators in the stage.
+    public static final String KEY_OF_OP_STATS_CACHE_MAX = "pinot.server.query.op.stats.cache.max";
+    public static final int DEFAULT_OF_OP_STATS_CACHE_MAX = 1000;
+
+    /// Max time to keep the op stats in the cache.
+    public static final String KEY_OF_OP_STATS_CACHE_EXPIRE_MS = "pinot.server.query.op.stats.cache.secs";
+    public static final int DEFAULT_OF_OP_STATS_CACHE_EXPIRE_MS = 10 * 60 * 1000;
```

Comment:
The variable name says MS and the default value seems to be 10 minutes but the config name says `secs` instead of `ms`?

Reply:
Fixed